pkg/oc/experimental/ipfailover/ipfailover.go

```diff
@@ -98,6 +98,7 @@ func NewCmdIPFailoverConfig(f *clientcmd.Factory, parentName, name string, out,
 	cmd.Flags().BoolVar(&options.Create, "create", options.Create, "If true, create the configuration if it does not exist.")
 
 	cmd.Flags().StringVar(&options.VirtualIPs, "virtual-ips", "", "A set of virtual IP ranges and/or addresses that the routers bind and serve on and provide IP failover capability for.")
+	cmd.Flags().StringVar(&options.VIPGroups, "virtual-ip-groups", "", "Number of groups to create for VRRP, if not set a group will be created for each virtual ip given on --virtual-ips.")
```

Comment:
You could set the default value in the command line option. 
If you set it in the script then the user will not know our default.